dask_ml/model_selection/_hyperband.py

```diff
@@ -403,10 +461,9 @@ async def _fit(self, X, y, **fit_params):
         )
         SHAs = {b: SHA for b, SHA in zip(_brackets_ids, _SHAs)}
 
-        # This for-loop rename estimator IDs and pulls out wall times
-        key = "bracket={}-{}".format
-        for b, SHA in SHAs.items():
+        def _rename_model_ids(SHA, b, key=lambda x: x):
```

Comment:
I've we're only using this once, I'd prefer to just do it inline rather than as a closure.